### PR TITLE
Added PAssert, for static importing

### PR DESCRIPTION
**Pull request by markijbema**
This should enable Passert(....) instead of PAssert.That(...)

### CODE REVIEW

File: ExpressionToCodeLib/PAssert.cs [hunk 1]
```diff
@@ -14,6 +14,11 @@ public static void IsTrue(Expression<Func<bool>> assertion)
             That(assertion);
         }
 
+        public static void PAssert(Expression<Func<bool>> assertion, string msg = null)
+        {
+        	That(assertion, msg);
+        }
+
```
Comment (EamonNerbonne) [inline ExpressionToCodeLib/PAssert.cs]: This class already contains methods like `That` and `IsTrue` which might be a little weird in a static import class - perhaps a separate class especially for static imports would be better?  Or is that too undiscoverable?

Comment (markijbema) [inline ExpressionToCodeLib/PAssert.cs]: Good point; maybe PAssert.Static or something, and add it to the docs.

**Comment by EamonNerbonne** [comment]
any update?

File: ExpressionToCodeLib/PAssert.cs (hunk 1 above)
Comment (EamonNerbonne) [inline ExpressionToCodeLib/PAssert.cs]: perhaps `PAssertStaticImports` or something similarly unique?

**Pull request merged**